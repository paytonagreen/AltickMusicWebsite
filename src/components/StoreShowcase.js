import React from 'react';
import { Link } from 'react-router-dom';

const StoreShowcase = () => {
  return (
    <section id="showcase">
        <div className="showcase-content">
            <h3><span className="secondary-text">Store coming soon!</span></h3>
            <p className="tertiary-text">In the meantime, check out our Reverb page!</p>
            <a href="http://reverb.com/shop/altick-music-co" className="btn-blue">Our Reverb.com Page</a>
        </div>
    </section>
  )
};

export default StoreShowcase;