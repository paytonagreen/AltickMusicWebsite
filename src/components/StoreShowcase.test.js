import React from 'react';
import { shallow } from 'enzyme'

import StoreShowcase from './StoreShowcase';

describe('StoreShowcase', () => {
  const storeShowcase = shallow(<StoreShowcase/>);

  it ('renders correctly', () => {
    expect(storeShowcase).toMatchSnapshot();
  });

  it('links to Reverb', () => {
    beforeEach(() => {
      storeShowcase.find('btn-blue').simulate('click');
    })
    expect(storeShowcase.find('a').props().href).toBe('http://reverb.com/shop/altick-music-co')
  })
});