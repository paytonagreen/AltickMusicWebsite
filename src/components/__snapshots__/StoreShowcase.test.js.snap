// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoreShowcase renders correctly 1`] = `
<section
  id="showcase"
>
  <div
    className="showcase-content"
  >
    <h3>
      <span
        className="secondary-text"
      >
        Store coming soon!
      </span>
    </h3>
    <p
      className="tertiary-text"
    >
      In the meantime, check out our Reverb page!
    </p>
    <a
      className="btn-blue"
      href="http://reverb.com/shop/altick-music-co"
    >
      Our Reverb.com Page
    </a>
  </div>
</section>
`;
